packages/algob/src/internal/deployer.ts

```diff
@@ -274,7 +274,7 @@ class DeployerBasicMode {
 	sendAndWait(
```

Comment:
lets update the method description about the return type

Reply:
sure